CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [0.2.1] - 2019-11-17
+
+### Added
+
+- Added new `$in_list` parameter to the `PrefixAllGlobalsSniff::process_variable_assignment()` method, so that the method is compatible with the upstream WPCS method (changed in WPCS 2.2.0).
+- Travis: Minor build script tweaks.
+
+### Changed
+
+- Travis: Changed the WPCS branch to `dev-develop` in `env` settings.
```

Comment:
This one is now covered by the `Travis: Minor build script tweaks.` and doesn't need a separate entry.

Though the `Travis: Minor build script tweaks.` should probably be moved to the `Changed` section.